build.gradle

```diff
@@ -63,13 +63,16 @@ compileTestKotlin { kotlinOptions.jvmTarget = sourceCompatibility }
 
 repositories {
     mavenCentral()
+    jcenter()
 }
                        
 dependencies {
 //    testImplementation 'io.cucumber:cucumber-java:4.2.6'
 //    testImplementation 'io.cucumber:cucumber-junit:4.2.6'
     testImplementation 'info.cukes:cucumber-java:1.2.4'
     testImplementation 'info.cukes:cucumber-junit:1.2.4'
+    // https://mvnrepository.com/artifact/khttp/khttp
+    compile 'khttp:khttp:1.0.0'
```

Comment:
As far as I understand the library is quite exotic, better to use smth more widespread + with mit/bsd/apache license